weatherman.py

```diff
@@ -0,0 +1,177 @@
+import os
+import sys
+import argparse
+import datetime
+import glob
+import csv
+
+class DayReport:
```

Comment:
please read pep-8 conventions, there should 2 spaces b/w imports and class/function.